lib/util/exec/index.ts

```diff
@@ -144,12 +150,12 @@ export async function exec(
   opts: ExecOptions = {}
 ): Promise<ExecResult> {
   const { docker } = opts;
-  const { dockerChildPrefix } = GlobalConfig.get();
+  const { dockerChildPrefix = 'renovate_' } = GlobalConfig.get();
```

Comment:
```suggestion
  const dockerChildPrefix = GlobalConfig.get('dockerChildPrefix');
```

Better type inference. Why a default value here?